packages/graphql/src/mutations/_txHelper.js

```diff
@@ -26,44 +26,54 @@ export default function txHelper({
   mutation,
   onConfirmation,
   onReceipt,
-  from
+  from,
```

Comment:
nit: since txHelper is going to a be a central function in the system, would be good to explicitly document the various arguments it takes as opposed to just relying on arg names.